test/powershell/engine/ResourceValidation/MPCUResource.Tests.ps1

```diff
@@ -0,0 +1,17 @@
+. "$psscriptroot/TestRunner.ps1"

```

Comment:
Since the file for management dll is `ManagementCommandsResources.Tests.ps1`, Maybe change `MPCUResource.Tests.ps1` to `Utility.Resource.Tests.ps1`?

Reply:
will change